packages/jni-swig-stub/src/main/jni/realm_api_helpers.cpp

```diff
@@ -533,6 +533,7 @@ void set_log_callback(realm_sync_client_config_t* sync_client_config, jobject lo
                                                                                "log",
                                                                                "(SLjava/lang/String;)V");
                                                   jenv->CallVoidMethod(log_callback, log_method, level, to_jstring(jenv, message));
+                                                  jni_check_exception(jenv);
```

Comment:
Could we write a test that checks this?

Reply:
I think it is quite hard to write some tests that verifies that the logger writes the correct thing, but I guess we could write some tests just to ensure that we do not crash for string containing formatting specifiers, etc. But could consider to do this in another PR.

Reply:
I would vote for just merging this, since it is blocking other things and create an issue for tests for this.